src/sentry/south_migrations/0454_auto__add_field_savedsearch_is_global__chg_field_savedsearch_project__.py

```diff
@@ -0,0 +1,1262 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    # Flag to indicate if this migration is too risky
+    # to run online and needs to be coordinated for offline
+    is_dangerous = False
+
+    def forwards(self, orm):
+        # Adding field 'SavedSearch.is_global'
+        db.add_column('sentry_savedsearch', 'is_global',
+                      self.gf('django.db.models.fields.BooleanField')(default=False),
+                      keep_default=False)
+
+        # Changing field 'SavedSearch.project'
+        db.alter_column(
+            'sentry_savedsearch',
+            'project_id',
+            self.gf('sentry.db.models.fields.foreignkey.FlexibleForeignKey')(
+                to=orm['sentry.Project'],
+                null=True))
```

Comment:
can you print out the sql for this migration to make sure it looks sane? this seems like the type of thing that south might overly complicate

Reply:
```
ALTER TABLE "sentry_savedsearch" DROP CONSTRAINT "project_id_refs_id_b18120e7" []
ALTER TABLE "sentry_savedsearch" ALTER COLUMN "project_id" TYPE bigint, ALTER COLUMN "project_id" DROP NOT NULL, ALTER COLUMN "project_id" DROP DEFAULT; []
ALTER TABLE "sentry_savedsearch" ADD CONSTRAINT "project_id_refs_id_b18120e7" FOREIGN KEY ("project_id") REFERENCES "sentry_project" ("id") DEFERRABLE INITIALLY DEFERRED;
```

Dang, I guess in general it's better to write the SQL directly? All I need is the drop not null...

Reply:
yeah in this case it looks like it. you can just write the sql for postgres and let south handle the other db types. in case you need an example:
https://github.com/getsentry/sentry/blob/master/src/sentry/south_migrations/0285_auto__chg_field_release_project_id__chg_field_releasefile_project_id.py#L12-L31